detectors/inferred-services-detectors/POST_Detector_error_rate.sh

```diff
@@ -0,0 +1,90 @@
+curl --location 'https://api.us1.signalfx.com/v2/detector' \
```

Comment:
please chmod the *.sh files to something like 766 so anyone can read/write but only owner can execute. Currently script isn't executable without chmoding it.